File: library/src/main/java/com/pengrad/telegrambot/model/request/ShippingOption.java
package com.pengrad.telegrambot.model.request;

import java.io.Serializable;

/**
 * Stas Parshin
 * 24 May 2017
 */
public class ShippingOption implements Serializable {
    private final static long serialVersionUID = 0L;

    private String id, title;
    private LabeledPrice[] prices;

    public ShippingOption(String id, String title, LabeledPrice... prices) {
        this.id = id;
        this.title = title;
        this.prices = prices;
    }
}

File: core/src/main/kotlin/com/pengrad/telegrambot/model/request/ShippingOption.kt
package com.pengrad.telegrambot.model.request

import java.io.Serializable

/**
 * Stas Parshin
 * 24 May 2017
 */
class ShippingOption(
        private val id: String,
        private val title: String,
        private vararg val prices: LabeledPrice
) : Serializable {
    companion object {
        @JvmStatic
        private val serialVersionUID = 0L

    }
}
